.clj-kondo/imports/org.cyverse/clojure-commons/clojure_commons/config.clj

```diff
@@ -0,0 +1,135 @@
+(ns clojure-commons.config)
```

Comment:
This namespace contains pared-down versions of our configuration macros that are intended solely to allow ling checks to work correctly for the config macros.